script/setup-dev

```diff
@@ -22,6 +22,7 @@ NODE_VERSION=$(cat "${DIR}/../.node-version")
 
 if command node -v 2>/dev/null | grep -q "${NODE_VERSION}"; then
   echo "Node version ${NODE_VERSION} already installed"
+  sudo corepack enable
```

Comment:
Rather than duplicating this line maybe do the install stuff in an `else` and have the existing `sudo corepack enable` after the `fi`?

Reply:
I like this suggestion. It makes it easier to understand we always want `corepack` to run, even if Node is already installed.